linux_os/guide/services/ntp/chronyd_no_chronyc_network/ignition/shared.yml

```diff
@@ -0,0 +1,16 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+          source: data:,cmdport%200%0A
+        filesystem: root
+        mode: 0644
+        path: /etc/chrony.conf
+        append: true
```

Comment:
This approach is nice, but unfortunately it won't work because Machine operator does not support the append parameter. You will need to rewrite the rule so that all rules connected with this config file are remediated.

Reply:
I noticed that as well yesterday after I submitted - will update